common/enclave/cc_data.cpp

```diff
@@ -30,12 +30,13 @@ bool cc_data::generate_keys()
 {
     try
     {
-        signature_key_.Generate();  // private key
-        verification_key_ = signature_key_.GetPublicKey();
-        decryption_key_.Generate();  // private key
-        encryption_key_ = decryption_key_.GetPublicKey();
-        cc_decryption_key_.Generate();  // private key
-        cc_encryption_key_ = cc_decryption_key_.GetPublicKey();
+        signature_key_.Generate();                          // enclave_sk, private signing key
+        verification_key_ = signature_key_.GetPublicKey();  // enclave_vk, public verifying key
+        decryption_key_.Generate();                         // enclave_dk, private decryption key
+        encryption_key_ = decryption_key_.GetPublicKey();   // enclave_ek, public encryption key
+        cc_decryption_key_.Generate();                      // chaincode_dk, private decryption key
+        cc_encryption_key_ =
+            cc_decryption_key_.GetPublicKey();  // chaincode_ek, public encryption key
 
```

Comment:
Don't have to do it in this PR but we also need the chaincode state encryption key which presumably also would be created here? So a todo might be worthwhile ...

```suggestion
        // TODO: create chaincode state encryption key ...
```